MapboxNavigation/NavigationMapView.swift

```diff
@@ -1085,12 +1090,13 @@ open class NavigationMapView: MGLMapView, UIGestureRecognizerDelegate {
             return
         }
         
-        let cam = self.camera
-        cam.pitch = 0
-        cam.heading = 0
+        let currentCamera = self.camera
+        currentCamera.pitch = 0
+        currentCamera.heading = 0
+        
+        let newCamera = camera(currentCamera, fitting: line, edgePadding: padding)
```

Comment:
This is equivalent to calling `cameraThatFits(_:direction:edgePadding:)`, since we’re forcing the pitch to be 0° anyways.